src/main/views/includes/support-links.njk

```diff
@@ -1,3 +1,21 @@
+<ul>
+  <li>
+    <a href="/cookies">Cookies</a>
+  </li>
+
+  <li>
+    <a href="/privacy-policy">Privacy Policy</a>
+  </li>
+
+    <li>
```

Comment:
reformat this section please, spacing is all over the place

Reply:
👍